### DISCUSSION TITLE
Custom Styles CSS

### DISCUSSION THREAD

**Discussion by triton-nereide** [UnchartedBull/OctoDash#3746] [General]
I ve got a question concerning custom-styles.css. I´ve read in the readme that I can upload a new custom-styles css file to octodash, via the normal file upload in octoprint. Some questions:
- do I need to rename the current custom-style.css to something like custom-styles.css.old or .orig bevor uploading a new one?
- when I upload the new one via file upload, I assume I need to move the file from upload directory to octodash directory?
Is there some place where the process is explained in detail? I don´t want to mess up something in my installation...
Does anybody have testet the custom-styles.css "square" from the skinns section, how does it look? How is it different from standard? This is the only skon that has no screenshots available online...

**Comment by jneilliii** [comment]
This is assuming you have the companion plugin installed and you do not need to do anything but upload and then restart octodash.

**Reply by triton-nereide** [reply]
Ah, O.K., I will try! I have installed Octodash Companion...

Thank you again, jneilliii

**Reply by triton-nereide** [reply]
Is there a way to backup the original "custom-Styles.css" in case I don´t like the outcome?

**Reply by jneilliii** [reply]
you'd have to SSH to the pi and manually copy the file. you can always go back to the default or one of the other themes though. 

`cp ~/.config/octodash/custom-styles.css ~/.config/octodash/custom-styles.css.bak`

**Reply by triton-nereide** [reply]
Thank you very much, jenelliii, everything worked out as you said!
